force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -163,9 +172,13 @@ public with sharing class ClassesObjectsMethods {
      */
     public static Integer generateRandomNumber() {
         // Use Math class to generate number between 50 and 100
+        Integer r = (Integer)(Math.random() * 10);
+        if (r < 50) {
+            r+= 50;
+        } 
 
         // Return the random integer
-        return null; // Replace null with the variable you used to store the result
+        return r ; // Replace null with the variable you used to store the result
```

Comment:
Remember, the goal here  was to generate numbers between 50 and 100, not 50 and 59